rclcpp/minimal_service/not_composable.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include "rclcpp/rclcpp.hpp"
+#include "examples_rclcpp_minimal_service/srv/add_two_ints.hpp"
+
+using Service = examples_rclcpp_minimal_service::srv::AddTwoInts;
```

Comment:
I'd recommend two things here:

- use `example_interfaces/srv/AddTwoInts`
- do `using example_interfaces::srv::AddTwoInts;` and replace `Service` with `AddTwoInts` throughout

I don't know if there was a reason for creating the service locally, but every message and/or service we have to build on the farm is lots of extra time. As for illustrating how to make your own service, perhaps we could just leave the machinery for that commented out or just refer to the `example_interfaces` package, it is quite simple already:

https://github.com/ros2/examples/blob/master/example_interfaces/CMakeLists.txt

I'm less passionate about replacing `Service` with `AddTwoInts`, but I think it does give more meaning to each place it is used (in the callback and at the creation of the service). It gets really important if the user is going to create two services in the same file.